src/lsptoolshost/server/roslynLanguageServer.ts

```diff
@@ -284,6 +282,8 @@ export class RoslynLanguageServer {
                 protocol2Code: UriConverter.deserialize,
             },
             middleware: {
+                provideDiagnostics,
+                provideWorkspaceDiagnostics,
```

Comment:
Can/should this be put in the razorLanguageServer too? Presumably we'll now be relaying the information diagnostics back from Roslyn un-changed, so may as well honour the same setting?

Reply:
Seems reasonable. =)